lib/mongo/collection/view/readable.rb

```diff
@@ -549,22 +549,26 @@ def server_selector
         end
 
         def parallel_scan(cursor_count, options = {})
-          session = client.send(:get_session, @options)
+          if options[:session]
+            session = client.send(:get_session, @options)
+          else
+            session = nil
+          end
           server = server_selector.select_server(cluster)
           cmd = Operation::ParallelScan.new({
                   :coll_name => collection.name,
                   :db_name => database.name,
                   :cursor_count => cursor_count,
                   :read_concern => collection.read_concern,
-                  :session => session
+                  :session => session,
                 }.merge!(options))
           cmd.execute(server).cursor_ids.map do |cursor_id|
             result = if server.features.find_command_enabled?
                        Operation::GetMore.new({
-                         :selector => {:getMore => cursor_id,
+                          :selector => {:getMore => cursor_id,
```

Comment:
It's hard to tell from the Github diff viewer, but is this being changed from 2 to 3 space indentation (relative to the beginning of `Operation::GetMore`)?